tests/fixtures/nordic-x-shapes.json

```diff
@@ -0,0 +1,586 @@
+{
```

Comment:
Better to name fixtures by the structure they have (like MR x CAT or whatever) and any other relevant characteristics. "nordic" means nothing to me and does not help me understand what is distinctive about this test case. Also, when named by structure, it helps locate existing fixtures that can be reused for other tests.